runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ExternalTest.java

```diff
@@ -86,26 +84,27 @@ public static void tearDown() {
   @Test
   @Category({ValidatesRunner.class, UsesCrossLanguageTransforms.class})
   public void expandSingleTest() {
-    PCollection<Integer> col =
+    PCollection<String> col =
         testPipeline
-            .apply(Create.of(1, 2, 3))
+            .apply(Create.of("1", "2", "3"))
```

Comment:
Was there any reason for changing the type here (from Integer to string) ?

Reply:
The test is modified as close to Python external_test. It was just adding 1 but now concatenating `Simple(%s)`.